### ISSUE TITLE
Sometimes the wearables change in our avatar is not being seen by the other users

### ISSUE THREAD

**Issue by sandrade-dcl** [decentraland/unity-renderer#1997]
Sometimes, when we equip a wearable from our backpack, our new aspect is not being notified to the other users.

**Steps to reproduce:**
1. Enter the world with 2 browsers with different users (for example one logged with the Metamask wallet and the other one as guest).
2. Go to the same places with both users in order to make both users see each other.
3. From Metamask user: go to the backpack and equip some wearable.
4. From Guest user: you should see the visual change in the other avatar.
5. Repeat the process and you will notice that a lot of times hte visual update is not happening from the other user perspective.

We don't know if it could be something related to how we, from renderer, are filtering the sending of the information to kernel when we changes our avatar, or something related to how kernel/comms is propagating the changes along the users. It should be debugged.

**Comment by davidejensen** [comment]
I can't reproduce this issue.
I've tried many different things and it appears that the wearables are updated correctly every time.
I've tried in different realms, in crowded areas and in isolated areas, tried by changing one or many wearables, changing nft ones or default ones. Tried reloading the sessions and with different configs to have lower and higher frame rates, never managed to break it.
